src/Committee/CommitteeCreationCommandHandler.php

```diff
@@ -14,23 +14,30 @@ class CommitteeCreationCommandHandler
     private $factory;
     private $manager;
     private $mailer;
+    private $committeeManager;
 
     public function __construct(
         EventDispatcherInterface $dispatcher,
         CommitteeFactory $factory,
         ObjectManager $manager,
-        MailerService $mailer
+        MailerService $mailer,
+        CommitteeManager $committeeManager
     ) {
         $this->dispatcher = $dispatcher;
         $this->factory = $factory;
         $this->manager = $manager;
         $this->mailer = $mailer;
+        $this->committeeManager = $committeeManager;
     }
 
     public function handle(CommitteeCreationCommand $command): void
     {
         $adherent = $command->getAdherent();
         $committee = $this->factory->createFromCommitteeCreationCommand($command);
+        // Uploads an ID photo
+        if (null !== $command->getPhoto()) {
```

Comment:
Use `hasPhoto` or store the result on the first call like `null !== $photo = $command->getPhoto()`.
I suggest to either do that in the manager, or to pass it the committee and the photo.
The command must protect the "real" committee to be in an invalid state.